### PR TITLE
Add the Salary Survey and 2020 Community Update

### PR DESCRIPTION
**Pull request by ericholscher**
This is an initial attempt to ship our Community Update & Salary Survey.
It adds the Survey to the website,
and updates the Team pages where it's obvious that we need to update them.

This definitely needs some review and cleanup,
along with adding in the jobs data before we send it.

### CODE REVIEW

File: docs/blog/2020-community-update.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+   :author: Eric Holscher
+
+Write the Docs 2020 Community Update
+====================================
+
+Hey there documentarians!
+
+The newsletter team is taking September off as usual,
+but there's still plenty going on in the community.
+We're sending out a update to let you know about a few things we're working on while they take a much deserved break.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
We're sending out this update to let you know what else is going on behind the scenes on while they take a much deserved break.
```

File: docs/blog/2020-community-update.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+The talks were just announced for Prague, and the CFP is open for a little while longer for Australia & India.
+
+We also just recently wrapped up our first online Portland conference.
+We had around 600 people join us to make this another memorable event.
+The `videos <https://www.writethedocs.org/videos/portland/2020/>`_ are now online from that conference.
+
+Salary Survey
+-------------
+
+We were amazed by the response to our Salary Survey last year.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]: Do we wanna link to the results and say how many folks responded?

File: docs/blog/2020-community-update.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+We are excited to launch our new :doc:`2020 Salary Survey </surveys/salary-survey-sep-2020>`.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
We are excited to launch our new :doc:`2020 Salary Survey </surveys/salary-survey-sep-2020>`.
```

something something that isn't excited? "This year we're back with "?

File: docs/blog/2020-community-update.rst [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+It should take between 5 and 10 minutes to complete,
+so we hope that you'll `fill it out <https://2020-salary-survey.writethedocs.org/>`_.
+
+Community Shout Out
+-------------------
+
+The numbers of activities going on in the Write the Docs community grows each year.
+Most of these are things are organized and run primarily by volunteers,
+and we wanted to take time to show appreciation for these folks.
+We will certainly miss many of people doing great work around out community,
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
While we can't mention every member of our extremely welcoming community, 
```
maybe?

File: docs/blog/2020-community-update.rst [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+but we wanted to show appreciation for folks in particular who are working behind the scenes.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
we want to show particular appreciation for folks who are working behind the scenes.
```

File: docs/blog/2020-community-update.rst [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+Newsletter Team
+~~~~~~~~~~~~~~~
+
+First up, as seems fitting, is our Newsletter team.
+Y'all probably know Beth,
+who is our editor,
+but there is an entire group working behind the scenes to write the articles each month.
+They are responsible for making this newsletter happen each month:
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]: I'd like to reword this but don't have a helpful suggestion

File: docs/blog/2020-community-update.rst [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+* `Hillary Fraley <https://github.com/hillaryfraley>`_
+* `Jennifer Rondeau <https://twitter.com/bradamante>`_
+* `Claire Lundeby <https://twitter.com/clairelundeby>`_
+* `Elle Jones <https://twitter.com/party_parrot18>`_
+* `Kyla del Rosario <https://kyladelrosario.com/>`_
+
+Moderation Team
+~~~~~~~~~~~~~~~
+
+Next is our Slack moderators.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
Next are our Slack moderators.
```

File: docs/blog/2020-community-update.rst [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+* `Janine Chan <https://www.linkedin.com/in/janinechan/>`_
+* `Daniel Beck <https://twitter.com/ddbeck>`_
+* `Mike Jang <https://twitter.com/themikejang>`__
+* `Eric Holscher <https://twitter.com/ericholscher>`_
+* `Samuel Wright <https://twitter.com/plaindocs>`_
+
+Meetup Coordination Team
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are too many meetup organizers to list across our lovely community.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]: Link to meetup page

File: docs/blog/2020-community-update.rst [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+Y'all get to know them at each meetup that you go to,
+but behind the scenes there are folks who are doing coordination to make our meetup community stronger.
+That is the job of the meetup coordination team,
+and our meetups wouldn't be as useful without them:
+
+* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Margaret Eker <https://twitter.com/meker>`_
+* `Mike Jang <https://twitter.com/TheMikeJang>`_
+* `Swapnil Ogale <https://twitter.com/swapnilogale>`_ (APAC)
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]: Seems odd to put the APAC here

File: docs/blog/2020-community-update.rst [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+Support Team
+~~~~~~~~~~~~
+
+This team is responsible for answering all the various emails that get sent to us at Write the Docs.
+There have been 773 emails so far this year,
+so this is a huge task that is ongoing throughout the year.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
so this is a huge ongoing task.
```

File: docs/blog/2020-community-update.rst [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+.. TODO
+
+*To apply for these job, or post your own, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
+
+Thanks
+------
+
+Thanks again for subscribing to our newsletter and being a member of our community.
+We hope to see you in person soon at one of our events,
+or continue to see you online via this newsletter.
```
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
We hope to see you soon at one of our online events, on our slack, 
or continue to see you here via this newsletter.
```

File: docs/team.rst [hunk 12]
```diff
@@ -144,7 +129,28 @@ Roles
...
+* `Samuel Wright <https://twitter.com/plaindocs>`_
+* You?
+
+
+Guide Coordination Team
+-----------------------
+
+These are the folks who maintain https://www.writethedocs.org/guide/ and make sure it stays high quality.
+The site is built with Sphinx and hosted on Read the Docs,
+but
```
Comment (plaindocs) [inline docs/team.rst]:
```suggestion
The site is built with Sphinx and hosted on Read the Docs,
but
```

**Review by plaindocs — COMMENTED** [review]
Some nits. I don't have much polish to add today.

File: docs/blog/2020-community-update.rst (hunk 9 above)
Comment (ericholscher) [reply docs/blog/2020-community-update.rst]: Yea, just copied from the team page, removed.

**Comment by plaindocs** [comment]
Where are we at with pushing this out of the door? @writethedocs/core

File: docs/blog/2020-community-update.rst [hunk 13]
```diff
@@ -0,0 +1,116 @@
+.. post:: Sep 7, 2020
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs 2020 Community Update
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst]: Not sure why the "2020" is here, this feels like a mid-year update. IMO we can remove it and still get the message across

File: docs/blog/2020-community-update.rst [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+but there's still plenty going on in the community.
+This update covers what else is going on behind the scenes while they take a much deserved break.
+
+We're looking forward to seeing everyone online for our `Prague <http://www.writethedocs.org/conf/prague/2020/>`_ on October 18-20 in the CEST timezone.
+We're also getting excited about our upcoming `Australia & India <http://www.writethedocs.org/conf/australia/2020/>`_ conference on December 3-4 in the AEDT timezone.
+The talks were just announced for Prague, and the CFP is open for a little while longer for Australia & India.
+
+We also just recently wrapped up our first online Portland conference.
+We had around 600 people join us to make this another memorable event.
+The `videos <https://www.writethedocs.org/videos/portland/2020/>`_ are now online from that conference.
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst]:
```suggestion
The `videos <https://www.writethedocs.org/videos/portland/2020/>`_ from the conference are now available online.
```

File: docs/blog/2020-community-update.rst [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+We have now launched our :doc:`2020 Salary Survey <https://2020-salary-survey.writethedocs.org/>`.
+We were amazed by the response to our Salary Survey last year,
+with `650 responses <https://www.writethedocs.org/surveys/salary-survey/2019/>`_ and a lot of interesting data that came out of it.
+
+This annual survey, run by the Write the Docs community, aims to gather data about salaries for people working in documentation and related fields around the world. `Results of the inaugural survey <https://www.writethedocs.org/surveys/salary-survey/2019/>`_ are available on the our website.
+
+We hope that this data can help our community members determine what appropriate salary ranges are, and provide a benchmark for future employment-related decisions and negotiations.
+Additionally, we’re hoping the results spark discussion of employment-related trends and issues in our industry, including but not limited to the effects of the COVID-19 pandemic.
+
+The survey is broken up into 5 sections.
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst]:
```suggestion
The survey is divided to 5 sections.
```

File: docs/blog/2020-community-update.rst [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+It should take between 5 and 10 minutes to complete,
+so we hope that you'll `fill it out <https://2020-salary-survey.writethedocs.org/>`_.
+
+Community Shout Out
+-------------------
+
+The numbers of activities going on in the Write the Docs community grows each year.
+Most of these are things are organized and run primarily by volunteers,
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst]:
```suggestion
Most of these are organized and run primarily by volunteers,
```

File: docs/blog/2020-community-update.rst [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+and we wanted to take time to show appreciation for these folks.
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst]:
```suggestion
and we wanted to take the time to show appreciation for these folks.
```

File: docs/blog/2020-community-update.rst [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+~~~~~~~~~~~~
+
+This team is responsible for answering all the various emails that get sent to us at Write the Docs.
+There have been 773 emails so far this year,
+so this is a huge ongoing task done by these folks:
+
+* `Sasha Romijn <https://twitter.com/mxsash>`_
+* `Samuel Wright <https://twitter.com/plaindocs>`_
+
+Featured jobs
```
Comment (thatdocslady) [inline docs/blog/2020-community-update.rst:100]: Anything that should go here?

**Review by thatdocslady — COMMENTED** [review]
A few nit-picks but otherwise LGTM

File: docs/blog/2020-community-update.rst (hunk 13 above)
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
Write the Docs Community Update
```

File: docs/blog/2020-community-update.rst (hunk 15 above)
Comment (plaindocs) [inline docs/blog/2020-community-update.rst]:
```suggestion
The survey is divided into 5 sections.
```

Comment (plaindocs) [reply docs/blog/2020-community-update.rst]: @thatdocslady :-p

File: docs/blog/2020-community-update.rst (hunk 18 above)
Comment (plaindocs) [reply docs/blog/2020-community-update.rst:100]: yes

**Review by plaindocs — COMMENTED** [review]
- [x] Get featured jobs

File: docs/blog/2020-community-update.rst [hunk 19]
```diff
@@ -17,7 +17,7 @@ The talks were just announced for Prague, and the CFP is open for a little while
```
Comment (swapnilogale) [inline docs/blog/2020-community-update.rst:17]: CFPs closed last night (14 Sept) and we will be aiming to announce speakers by end of Sept, with the schedule published mid-Oct.

Comment (plaindocs) [reply docs/blog/2020-community-update.rst:17]: Good catch mate

**Review by mxsasha — APPROVED** [review]

**Pull request merged**